python/pyspark/sql/utils.py

```diff
@@ -118,7 +118,8 @@ def require_minimum_pandas_version():
     from distutils.version import LooseVersion
     import pandas
     if LooseVersion(pandas.__version__) < LooseVersion('0.19.2'):
-        raise ImportError("Pandas >= 0.19.2 must be installed on calling Python process")
+        raise ImportError("Pandas >= 0.19.2 must be installed on calling Python process: %s"
```

Comment:
Maybe, we could make it a bit better like .. `.. on calling Python process; however, your version was %s.`.

Reply:
Sounds good. I'll update them. Thanks!